owncloudApp/src/main/res/values/strings.xml

```diff
@@ -269,6 +269,9 @@
     <string name="biometric_not_enrolled">Register at least one biometric to use this feature</string>
     <string name="biometric_prompt_title">Biometric log in</string>
     <string name="biometric_prompt_subtitle">Log in using your biometric credential</string>
+    <string name="biometric_dialog_title">Do you want to additionally activate biometric security?</string>
+    <string name="biometric_dialog_no_option">No</string>
+    <string name="biometric_dialog_yes_option">Yes</string>
```

Comment:
I think there are some default strings for this, something like `common_yes` and `common_no` if I'm not wrong 🤔

Reply:
Done